AMBuildScript

```diff
@@ -254,11 +266,10 @@ class MMSConfig(object):
         cxx.linkflags += ['-lgcc_eh']
     elif builder.target.platform == 'mac':
       cxx.defines += ['OSX', '_OSX', 'POSIX']
-      cxx.cflags += ['-mmacosx-version-min=10.5']
+      cxx.cflags += ['-mmacosx-version-min=10.9']
       cxx.linkflags += [
-        '-mmacosx-version-min=10.5',
-        '-arch', 'i386',
-        '-lstdc++',
+        '-mmacosx-version-min=10.9',
```

Comment:
For what it's worth, I believe 10.7 is the min version for CS:GO (as well as TF2 the branch).

Reply:
I was preparing to release a new srcds binary that bumps the requirement for all games to 10.9. It's necessary to support automatic updates among other things, but I'll put it back to 10.5 for now until that's been released.